pyomo/solvers/plugins/solvers/GUROBI_RUN.py

```diff
@@ -86,6 +101,9 @@ def gurobi_run(model_file, warmstart_file, soln_file, mipgap, options, suffixes)
     # key is specified, so you have to stare at the
     # output to see if it was accepted.
     for key, value in options.items():
+        if key in ['ISV_Key']:  # ISV_Key not supported by Gurobi
+            continue
+
```

Comment:
@DFuller134 I do not see the need for this check (if key is in ['ISV_Key']). Can you explain?